### PR TITLE
Fixed the triangle for osx

### PR DESCRIPTION
**Pull request by olli-lauhaa**
The triangle renders as an unknown character in osx

### CODE REVIEW

**Comment by pgonzal** [comment]
Thanks for pointing this out.  However, the character you chose looks a little odd on Windows.  :-P

I'll see if I can boot up my Mac and find a Unicode character that looks good on both OS's.  It's unfortunate that standard web fonts don't render consistently across devices.

**Comment by JoshuaKGoldberg** [comment]
How about a simple 👋 instead?

**Comment by zulaica** [comment]
Perhaps the HTML character code would be more universal?
`&#9658;`

**Comment by octogonz** [comment]
I tested all three suggestions: ▶ and 👋 and &#9658;

@JoshuaKGoldberg's suggestion of 👋 seems actually more effective, so let's go with that.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ export class Statistics {
 
 We are developing a library package [@microsoft/tsdoc](https://www.npmjs.com/package/@microsoft/tsdoc) that provides an open source reference implementation of a parser.  Using this library is an easy way to ensure that your tool is 100% compatible with the standard.
 
-⏵ ***Give it a try!**  The <a target="_blank" href="https://microsoft.github.io/tsdoc/">TSDoc Playground</a> provides an interactive showcase of our parser!*
+&#x1F44B; ***Give it a try!**  The <a target="_blank" href="https://microsoft.github.io/tsdoc/">TSDoc Playground</a> provides an interactive showcase of our parser!*
```
Comment (sharwell) [inline README.md]: 💡 Could also use `:arrow_forward:` (▶️) which is a bit more readable in text view

**Pull request merged**